README.md

```diff
@@ -37,42 +37,81 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
   - [Overview](#overview)
   - [Table of contents](#table-of-contents)
   - [Installation](#installation)
-    - [Dependencies](#dependencies)
-    - [Deeprank2 Package](#deeprank2-package)
-    - [Test installation](#test-installation)
+    - [Dockerfile](#dockerfile)
+    - [Non-pythonic dependencies](#non-pythonic-dependencies)
+    - [Pythonic dependencies](#pythonic-dependencies)
+      - [Test installation](#test-installation)
     - [Contributing](#contributing)
-    - [Data generation](#data-generation)
-    - [Datasets](#datasets)
-      - [GraphDataset](#graphdataset)
-      - [GridDataset](#griddataset)
-    - [Training](#training)
+  - [Data generation](#data-generation)
+  - [Datasets](#datasets)
+    - [GraphDataset](#graphdataset)
+    - [GridDataset](#griddataset)
+  - [Training](#training)
   - [Computational performances](#computational-performances)
   - [Package development](#package-development)
 
 ## Installation
 
-The package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
+Note that the package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
 
-### Dependencies
+### Dockerfile
 
-Before installing deeprank2 you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.10 installed. The following dependency installation instructions are updated as of 14/09/2023, but in case of issues during installation always refer to the official documentation which is linked below:
+In order to try out the package without worrying about your OS and without the need of installing all the required dependencies, we created a `Dockerfile` that can be used for taking care of everything in a suitable container. After having cloned the repository and installed [Docker](https://docs.docker.com/engine/install/), run the following commands from the root of the repository.
```

Comment:
what are the downsides of this, if any? Why wouldn't people always do this rather than go through the individual steps described in next section?

Reply:
For me, running everything on the docker container was much much slower. I've asked Djura why this could be, I was imagining that for reasons the resources available in a container are not the same that you have on your machine.

Reply:
We should add a comment about when a user should/would want to install it in which of the two ways. Maybe local installation would really only be for developers, if the docker turns out to be equally fast, etc.

Reply:
For real use cases, when a user needs to process thousands of data, I don't see any easy way of making use of the docker container (they will need to submit jobs to the machines of a supercomputer cluster, as we did in Snellius). For now, I'd just stick to the advice to use the docker option for trying out the package, otherwise the "normal" installation. If and when we get a wider user base, we can always improve the instructions and investigate the options.

OT: that's also the reason why I don't want to call the "normal" installation section "local" (since it's not always the case, for example, if you want to install the package via SSH).

Reply:
I opened an issue about this discussion (#534 )